### PR TITLE
Update Cloudformation and Riff-Raff for Amigo Build

### PR DESCRIPTION
**Pull request by Calum-Campbell**
- Update Cloudformation to use new locations of dist and conf files in S3
- Use Amigo Bake in cloudformation and riff raff deploy

### CODE REVIEW

File: cloudformation/identity-admin-api.yaml [hunk 1]
```diff
@@ -137,7 +137,9 @@ Resources:
           Statement:
           - Effect: Allow
             Action: s3:GetObject
-            Resource: arn:aws:s3:::gu-identity-*/*
+            Resource:
+            - arn:aws:s3:::identity-artifacts/*
```
Comment (jfsoul) [inline cloudformation/identity-admin-api.yaml]: I didn't pick up on this on the other apps we have already done - but could we restrict this permission further to `:::identity-artifacts/identity-admin-api/*`? (or even `:::identity-artifacts/${Stage}/${App}/*`)

Comment (jfsoul) [reply cloudformation/identity-admin-api.yaml]: the same for the config bucket

File: cloudformation/identity-admin-api.yaml [hunk 2]
```diff
@@ -266,29 +268,45 @@ Resources:
       UserData:
         Fn::Base64:
           !Sub |
-            #!/bin/bash -ev
-  
-            source /identity-functions.sh
+
+            # /etc/gu/install_vars
             source /set-env.sh
```
Comment (jfsoul) [inline cloudformation/identity-admin-api.yaml]: do we still need `set-env.sh`?

File: cloudformation/identity-admin-api.yaml [hunk 3]
```diff
@@ -266,29 +268,45 @@ Resources:
       UserData:
         Fn::Base64:
           !Sub |
-            #!/bin/bash -ev
-  
-            source /identity-functions.sh
+
+            # /etc/gu/install_vars
```
Comment (jfsoul) [inline cloudformation/identity-admin-api.yaml]: Remove this comment and add the `#!/bin/bash -ev` back in?

File: cloudformation/identity-admin-api.yaml (hunk 1 above)
Comment (Calum-Campbell) [reply cloudformation/identity-admin-api.yaml]: Yep seems sensible.

File: cloudformation/identity-admin-api.yaml (hunk 3 above)
Comment (Calum-Campbell) [reply cloudformation/identity-admin-api.yaml]: 👍

File: cloudformation/identity-admin-api.yaml [hunk 4]
```diff
@@ -137,7 +137,9 @@ Resources:
           Statement:
           - Effect: Allow
             Action: s3:GetObject
-            Resource: arn:aws:s3:::gu-identity-*/*
+            Resource:
+            - arn:aws:s3:::identity-artifacts/${Stage}/${App}/*
```
Comment (jfsoul) [inline cloudformation/identity-admin-api.yaml]: Don't forget the `!Sub`!

File: cloudformation/identity-admin-api.yaml (hunk 2 above)
Comment (Calum-Campbell) [reply cloudformation/identity-admin-api.yaml]: Nope, cheers.

**Pull request merged**